draft-ietf-quic-http.md

```diff
@@ -632,8 +643,12 @@ The following settings are defined in HTTP/QUIC:
   SETTINGS_HEADER_TABLE_SIZE (0x1):
   : An integer with a maximum value of 2^32 - 1.  This value MUST be zero.
 
-  SETTINGS_ENABLE_PUSH (0x2):
-  : Transmitted as a Boolean
+  SETTINGS_MAX_PUSH_ID (0x2):
+  : An integer with a maximum value of 2^32 - 1.  This determines the initial
+    maximum Push ID that a server can use.  This setting only has meaning when
+    set by a client; a server MUST NOT provide a value for this setting.  A
+    client MUST treat receipt of this setting from a server as a connection
+    error of type HTTP_MALFORMED_SETTINGS.
```

Comment:
Can this be 0? I assume so, since there's no other way to disable PUSH now. Can you state this explicitly here (that a client can disable push by setting this value to 0)?